server/config/config.exs

```diff
@@ -42,7 +42,8 @@ config :esbuild,
       ~w(js/app.js --bundle --target=esnext --outdir=../priv/static/assets --external:/fonts/* --external:/images/*),
     cd: Path.expand("../apps/ingest_web/assets", __DIR__),
     env: %{"NODE_PATH" => Path.expand("../deps", __DIR__)}
-  ]
```

Comment:
just put this on a new line with the comma and comment it out? because if someone downloads it now and doesn't put a cert in, even if they don't need it it will fail

#, cacerts_path: "PATH"